doc/tutorial.txt

```diff
@@ -422,6 +422,39 @@ following sequence:
 	"unlock" : "mutexA"
 }
 
+* barrier : String. Used as at least a pair where the name must match.
```

Comment:
Do we need to check that we have at least two users of a barrier while parsing the json file?
If not the only user can get stuck, can't it?